common/output_stream.cpp

```diff
@@ -279,6 +279,8 @@ std::string ToStringSpvDim(SpvDim dim) {
       return "Buffer";
     case SpvDimSubpassData:
       return "SubpassData";
+    case SpvDimTileImageDataEXT:
+      return "SpvDimTileImageDataEXT";
```

Comment:
I think generally the `Spv` prefix is dropped.

Reply:
> I think generally the `Spv` prefix is dropped.

Thanks.  Fixed.